### DISCUSSION TITLE
支持onedrive（世纪互联）的视频网页播放

### DISCUSSION THREAD

**Discussion by coppea** [AlistGo/alist#210] [Ideas]
世纪互联onedrive不支持视频预览，mp4也不支持，我感觉是artplay的原因，其他如onemanager，zfile就支持视频预览

希望支持或者更换其他视频插件

**Comment by xhofe** [comment]
- https://caniuse.com/hevc
- https://alist-doc.nn.ci/docs/common-question#%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E4%B8%8D%E4%BA%86

**Comment by coppea** [comment]
刚刚测试了一下，safari支持，Chrome就不支持
非H265视频

视频地址 https://bafybeiaxkimbnlfanxmltljxrxkskhj2hjf24qzj42nh3dfjsf5hl5qqne.ipfs.dweb.link/从单细胞到人类的进化.mp4

**Comment by coppea** [comment]
dplay支持的视频预览太少了，不是所有的MPEG-4都支持，我强烈介意换个视频播放插件

**Comment by xhofe** [comment]
> 刚刚测试了一下，safari支持，Chrome就不支持 非H265视频
> 
> 视频地址 [https://bafybeiaxkimbnlfanxmltljxrxkskhj2hjf24qzj42nh3dfjsf5hl5qqne.ipfs.dweb.link/从单细胞到人类的进化.mp4](https://bafybeiaxkimbnlfanxmltljxrxkskhj2hjf24qzj42nh3dfjsf5hl5qqne.ipfs.dweb.link/%E4%BB%8E%E5%8D%95%E7%BB%86%E8%83%9E%E5%88%B0%E4%BA%BA%E7%B1%BB%E7%9A%84%E8%BF%9B%E5%8C%96.mp4)

你这个视频resolution是0 x 0，本身就是有问题的，试了好几个h5的播放器，没一个有画面的。

**Comment by coppea** [comment]
确实，仔细查看了chrome文档，Chrome不支持mpeg-4 part2视频，基本只支持mpeg-4 part10的视频，也就是H.264的视频

我的这些视频都是很老的视频，基本不是H.264编码的
